sdk/core/core-http/test/policies/throttlingRetryPolicy.node.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import nock from "nock";
```

Comment:
we are using HttpMockFacade in defaultHttpClientTests.node.ts.  I wonder where it's possible to use HttpMockFacade to test this scenario without introducing new dev dep on nock.

Reply:
I'll try that.

Reply:
It says unmatched
![image](https://user-images.githubusercontent.com/10452642/123316093-5133a000-d4e1-11eb-8d47-30948cbc1932.png)
I'm unsure on where I went wrong.

Here is the branch - https://github.com/Azure/azure-sdk-for-js/compare/harshan/core-http/throttling-policy-try-HttpMockFacade?expand=1

Reply:
Removed nock finally